index.js

```diff
@@ -51,12 +56,16 @@ module.exports = {
     }
   },
   treeForStyles: function(){
-    var bootstrapTree = new Funnel(this.treeGenerator(path.join(this.bootstrapPath, 'stylesheets')), {
-      srcDir: '/',
+    var bootstrapTree = new Funnel(this.treeGenerator(path.join(this._bootstrapPath, 'stylesheets')), {
       destDir: '/app/styles'
     });
 
     return bootstrapTree;
-  }
+  },
 
+  treeForVendor: function() {
+    return new Funnel(this._bootstrapPath, {
```

Comment:
not sure if you need to funnel that with `_super`'s vendor tree, it might make it easier in case you have other vendor files in this addon. 

Here's an example: https://github.com/MiguelMadero/ember-monaco-editor/blob/master/index.js#L30:33